### PR TITLE
[NNC] Add cpu fusion gflag

### PR DESCRIPTION
**Pull request by eellison**

### CODE REVIEW

File: torch/csrc/jit/codegen/fuser/interface.cpp [hunk 1]
```diff
@@ -5,15 +5,26 @@
...
+
+bool initCpuFuserEnabled() {
+  return FLAGS_torch_jit_enable_cpu_fusion;
+}
+
+} // namespace
+
 // Note: CPU fusion is currently disabled due to test flakiness
-bool cpu_fuser_enabled = false;
+bool cpu_fuser_enabled = initCpuFuserEnabled();
```
Comment (Krovatkin) [inline torch/csrc/jit/codegen/fuser/interface.cpp:27]: @eellison did you have a chance to verify that `initCpuFuserEnabled();` returns either a value passed via an env variable or a cmd flag and not the default false value?

Comment (eellison) [reply torch/csrc/jit/codegen/fuser/interface.cpp:27]: i'm still not sure  how to do that 😓

Comment (eellison) [reply torch/csrc/jit/codegen/fuser/interface.cpp]: okay tested now

**Review by Krovatkin — APPROVED** [review]
:shipit:

**Review by bertmaher — APPROVED** [review]

**Pull request closed without merging**